theme/themes/pxt/globals/site.variables

```diff
@@ -18,6 +18,7 @@
 @grey:#505050;
 
 @greenTextColor: #107C10;
+@microsoftorange:#D83B01;
```

Comment:
No need to specify here. use the color directly. or rename to something more descriptive of it's use and place it at the top of tutorial.less